### PR TITLE
{vis}[GCCcore/11.3.0] TurboVNC v3.0.1

### PR DESCRIPTION
**Pull request by maxim-masterov**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/t/TurboVNC/TurboVNC-3.0.1-GCCcore-11.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    '11acc18ae10355c7c6816fad0a68ed55fbcb7a04efd30740c619a1d82ecbd4c2',  # TurboVNC-3.0.1_fix-libX11-libs.patch
+]
+
+builddependencies = [
+    ('binutils', '2.38'),
+    ('CMake', '3.23.1'),
+]
+
+dependencies = [
+    ('Java', '11', '', True),
```
Comment (jfgrimm) [inline easybuild/easyconfigs/t/TurboVNC/TurboVNC-3.0.1-GCCcore-11.3.0.eb:26]:
```suggestion
    ('Java', '11', '', SYSTEM),
```

File: easybuild/easyconfigs/t/TurboVNC/TurboVNC-3.0.1-GCCcore-11.3.0.eb [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+    ('X11', '20220504'),
+    ('pixman', '0.40.0'),
+    ('libjpeg-turbo', '2.1.3'),
+    ('Mesa', '22.0.3'),
+    ('libGLU', '9.0.2'),
+    ('libglvnd', '1.4.0'),
+    ('OpenSSL', '1.1', '', True),
```
Comment (jfgrimm) [inline easybuild/easyconfigs/t/TurboVNC/TurboVNC-3.0.1-GCCcore-11.3.0.eb:33]:
```suggestion
    ('OpenSSL', '1.1', '', SYSTEM),
```

File: easybuild/easyconfigs/t/TurboVNC/TurboVNC-3.0.1-GCCcore-11.3.0.eb [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+    ('pam-devel', 'libpam0g-dev'),
+]
+
+configopts = "-DTVNC_BUILDJAVA=0 -DTVNC_SYSTEMX11=1 "
+configopts += "-DX11_X11_LIB=$EBROOTX11/lib/libX11.%s " % SHLIB_EXT
+configopts += "-DX11_Xau_LIB=$EBROOTX11/lib/libXau.%s " % SHLIB_EXT
+configopts += "-DX11_SM_LIB=$EBROOTX11/lib/libSM.%s " % SHLIB_EXT
+configopts += "-DX11_ICE_LIB=$EBROOTX11/lib/libICE.%s " % SHLIB_EXT
+configopts += "-DX11_ICE_LIB=$EBROOTX11/lib/libXext.%s " % SHLIB_EXT
+configopts += "-DX11_ICE_LIB=$EBROOTX11/lib/libICE.%s " % SHLIB_EXT
```
Comment (jfgrimm) [inline easybuild/easyconfigs/t/TurboVNC/TurboVNC-3.0.1-GCCcore-11.3.0.eb:46]: this doesn't look right to me, we're re-defining `X11_ICE_LIB` twice?

**Review by jfgrimm — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/t/TurboVNC/TurboVNC-3.0.1-GCCcore-11.3.0.eb (hunk 3 above)
Comment (maxim-masterov) [reply easybuild/easyconfigs/t/TurboVNC/TurboVNC-3.0.1-GCCcore-11.3.0.eb:46]: @jfgrimm thanks for spotting it. I've removed the duplicate option.

Comment (jfgrimm) [reply easybuild/easyconfigs/t/TurboVNC/TurboVNC-3.0.1-GCCcore-11.3.0.eb]: you've removed the last duplicate, but still have:
```
-DX11_ICE_LIB=.../libICE.%s
-DX11_ICE_LIB=.../libXext.%s
```
the second of which should probably be defining a different variable (`X11_XEXT_LIB` or something like that)

Comment (maxim-masterov) [reply easybuild/easyconfigs/t/TurboVNC/TurboVNC-3.0.1-GCCcore-11.3.0.eb]: @jfgrimm missed it, thanks. done

**Review by boegel — COMMENTED** [review]
lgtm

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3118.skitty.os - Linux RHEL 8.4, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/87a2354d55083cb4f594c66e9fc03c6c for a full test report.

**Review by boegel — APPROVED** [review]
lgtm

**Comment by boegel** [comment]
Going in, thanks @maxim-masterov!

**Pull request merged**